src/urbit/app/chess.hoon

```diff
@@ -835,18 +835,66 @@
   ^-  (unit (unit cage))
   ?+  path  (on-peek:default path)
     ::
-    ::  .^(noun %gx /=chess=/game/~1996.2.16..10.00.00..0000/noun)
-    ::  read game info
-    ::  either active or archived
-    [%x %game @ta ~]
+    :: send back a list of chess-games
+    [%x %archive *]
+      ?+  t.t.path  (on-peek:default path)
```

Comment:
I didn't know you could lay out scry paths the way you're doing here, I like it.

Reply:
thank you!